BUILD.bazel

```diff
@@ -0,0 +1,38 @@
+"""Root configuration for Bazel build tool.
+"""
+
+# Allow the tsconfig.json file to be referenced by subpackages
+exports_files(["tsconfig.json"])
+
+# Provides the npm-installed dependencies to Bazel actions.
+filegroup(
+    name = "node_modules",
+    srcs = glob(
+        ["node_modules/**/*"],
+        # Exclude directories that commonly contain filenames which are
+        # illegal bazel labels
+        exclude = [
+            # e.g. node_modules/adm-zip/test/assets/attributes_test/New folder/hidden.txt
+            "node_modules/**/test/**",
+            # e.g. node_modules/xpath/docs/function resolvers.md
+            "node_modules/**/docs/**",
+        ],
+    ),
+    # Allow this targets to be referenced by anyone.
+    # Other repositories running nodejs binaries will need to
+    # reference our installed node_modules directory.
+    visibility = ["//visibility:public"],
+)
+
+# Temporary target to allow `bazel test ...`
+# TODO(alexeagle): remove as soon as there is any other test in the repo
```

Comment:
Actually I have a test in the NX setup branch: https://github.com/juristr/ngx-tabs-libdemo/blob/with-demoproj-setup/libs/ngx-tabs-lib/src/tabs.component.spec.ts

Reply:
it seems I started from the wrong branch - bazel should be most similar to how nx is setup.
at some point I think we should re-base bazel on that branch